LettersPalindrome.rb

```diff
@@ -0,0 +1,34 @@
+# Metodo de entrada 
+def input_data(message)
+    puts "Ingrese la palabra"
+    palabra = gets.chomp
+    message = "¿quieres seguir?"
+    calcula_palin(palabra) 
+end
+
+# metodo que calcula si la palabra es un palindromo
+def calcula_palin(palabra)  
```

Comment:
Let's make this more readeable, yes `reverse` does the job , you could make this work in one single line:
```
def palindrome?
  word.reverse == 'word' ? true : false
end
```
then you return a boolean to validate it